WireUI/Sources/WireMainNavigationUI/Models/MainConversationFilter.swift

```diff
@@ -16,8 +16,10 @@
 // along with this program. If not, see http://www.gnu.org/licenses/.
 //
 
-public enum MainConversationFilter: Sendable {
-    case favorites, groups, oneOnOne
+import Foundation
+
+public enum MainConversationFilter: Sendable, Equatable {
+    case favorites, groups, oneOnOne, folder(id: UUID, name: String)
 }
```

Comment:
Wouldn't it be possible to not include the details of the folder in the UI code?
Since we don't display any of that.
```suggestion
public enum MainConversationFilter: Sendable, Equatable {
    case favorites, groups, oneOnOne, folders
}
```

Reply:
When implementing this for iPad in https://github.com/wireapp/wire-ios/pull/2156 I couldn't work out a way to do this as `MainConversationFilter` is converted to a `ConversationFilter`.

However it is not necessary in `SidebarSelectableMenuItem.swift`. I will fix this.